schemas/2015-06-01/Microsoft.Cdn.json

```diff
@@ -0,0 +1,222 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.Cdn.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Cdn",
+    "description": "Microsoft Cdn Resource Types",
+    "resourceDefinitions": {
+        "profiles": {
+            "type": "object",
+            "description": "The CDN profile",
+            "properties": {
+                "type": {
+                    "enum": [ "Microsoft.Cdn/profiles" ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2015-06-01"
+                    ]
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "sku": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "someOf": [
```

Comment:
yes, someOf is a json keyword. Please not that oneOf doesn't work here because when I execute validateTemplates.js, I will get following errors:

Actual: [
    {
        "message": "Data is valid against more than one schema from \"oneOf\": i
ndices 0 and 1",
        "dataPath": "/properties/sku/name",
        "schemaPath": "/properties/properties/properties/sku/properties/name/one
Of"
    }
]

Anything need to be changed on the validation scripts?